README.md

```diff
@@ -46,3 +46,23 @@ or with `npx`:
 ```sh
 $ npx add-dependencies /home/user/project/package.json moment@2.0.0 react@16.8 redux eslint --dev
 ```
+
+or via nodejs
+
+```js
+const npmAdd = require('add-dependencies');
```

Comment:
You'd better describe imports separately. People usually just copy-paste example code snippets, and this line and line 62 will definitely lead to an error.